bin/wp-merge-to-cp.sh

```diff
@@ -0,0 +1,40 @@
+#/usr/bin/env bash
+
+# First parameter is your fork
+# Second parameter is the ticket merged
```

Comment:
Yes, there is no way around this extra step.

It's not possible to port whole tickets at once, because there's not a one-to-one correspondence between ticket numbers and commits.

What would we do when a ticket contains multiple commits?  What about when some of those commits are to one branch, and the others go to another branch?

What about when a commit says "See #xxxx" but doesn't actually fix the ticket?

What about reverts?

This is why it's important to work from **changeset** numbers rather than **ticket** numbers.

Reply:
Seems reasonable, I will work on change that.